src/examples/dropdowns/SelectSize.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Dropdown, Field, Menu, Item, Select, Label } from '@zendeskgarden/react-dropdowns';
+
+interface IItem {
+  label: string;
+  value: string;
+}
+
+const items = [
+  { label: 'Cactus', value: 'item-1' },
+  { label: 'Orchid', value: 'item-2' },
+  { label: 'Succulent', value: 'item-3' }
+];
+
+const StyledField = styled(Field)`
+  * {
+    /* stylelint-disable-next-line declaration-no-important */
+    appearance: none !important;
+  }
+`;
+
+const Example = () => {
+  const [selectedDefaultItem, setSelectedDefaultItem] = useState(items[0]);
+  const [selectedCompactItem, setSelectedCompactItem] = useState(items[0]);
+
+  return (
+    <Grid>
+      <Row>
+        <Col>
+          <Dropdown
+            selectedItem={selectedCompactItem}
+            onSelect={setSelectedCompactItem}
+            downshiftProps={{ itemToString: (item: IItem) => item && item.label }}
+          >
+            <StyledField>
+              <Label>Plants</Label>
+              <Select isCompact>{selectedCompactItem.label}</Select>
+            </StyledField>
+            <Menu isCompact>
+              {items.map(option => (
+                <Item key={option.value} value={option}>
+                  {option.label}
+                </Item>
+              ))}
+            </Menu>
+          </Dropdown>
+        </Col>
+        <Col>
```

Comment:
Recommend flipping column order so that the example order corresponds with the wording order: "Select can be default or compact in size."